lib/Ast.ml

```diff
@@ -2279,6 +2279,7 @@ end = struct
           ; _ }
       , _ ) ->
         false
+    | Exp {pexp_desc= Pexp_object _; _}, _ -> false
```

Comment:
Could you add a small comment explaining why we have this pattern ?

Reply:
Done in https://github.com/ocaml-ppx/ocamlformat/pull/1288/commits/ed75834457eb13065794a4806fff5074e36218a9.